angular_client/modules/core/src/app/index.js

```diff
@@ -95,8 +80,8 @@ angular.module("fielddbAngular", [
   document.addEventListener("logout", function() {
     fieldDBApp.bug("user has logged out, page will reload to clear state and take them to the welcome page.");
   }, false);
-  document.addEventListener("authenticate:fail", function() {
-    fieldDBApp.warn("user isn't able to see anything, show them the welcome page");
+  document.addEventListener("authenticateFail", function(reason) {
+    fieldDBApp.warn("user isn't able to see anything, show them the welcome page", reason);
```

Comment:
@jrwdunham why did you switch from authenticate:fail to authenticateFail should we follow this change? for now we switched to authenticateFail so that dative will  show us as logged in when we refresh and/or login